backend/src/models/api-key.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Entity,
+  Index,
+  Column,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+  CreateDateColumn,
+  BaseEntity,
+  ManyToOne
+} from 'typeorm';
+import { Organization, User } from './';
+
+@Entity()
+export class ApiKey extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn()
+  createdAt: Date;
+
+  @UpdateDateColumn()
+  updatedAt: Date;
+
+  @ManyToOne((type) => User, (user) => user.apiKeys, {
+    onDelete: 'CASCADE',
```

Comment:
I don't recall what this would mean -- would this mean that on deletion of a user the API key will be deleted (which is what we want) or on deletion of the API key the user will be deleted (which we don't want)?

Reply:
I think this means that when the user is deleted, the API key would also be deleted (judging by the fact that my account hasn't been deleted when I delete an API key)